Modules/_datetimemodule.c

```diff
@@ -660,28 +665,28 @@ static int
 check_time_args(int h, int m, int s, int us, int fold)
 {
     if (h < 0 || h > 23) {
-        PyErr_SetString(PyExc_ValueError,
-                        "hour must be in 0..23");
+        PyErr_Format(PyExc_ValueError,
+                    "hour must be in 0..23, but got %i", h);
```

Comment:
Nit: Please fix the alignment of the continuation lines so the double quote nicely aligns with the above line. Ditto for the four subsequent changes.

```suggestion
                     "hour must be in 0..23, but got %i", h);
```

Reply:
The subsequent four changes were not addressed.